deliver/lib/deliver/upload_metadata.rb

```diff
@@ -184,7 +184,7 @@ def load_from_filesystem(options)
           next unless File.exist?(path)
 
           UI.message("Loading '#{path}'...")
-          options[key] ||= {}
+          options[key] = {}
```

Comment:
@parleer Sorry, I can't understand your situation. Could you explain in detail?
What the value you passed to `name:`?